src/FlaUI.Core.UnitTests/Logging/AbstractBaseLoggerTests.cs

```diff
@@ -0,0 +1,300 @@
+﻿using FlaUI.Core.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Globalization;
+
+namespace FlaUI.Core.UnitTests.Logging
```

Comment:
Ok. You can adjust it then. Probably only for the UnitTest assembly and not for the UITest one since that tests multiple other assemblies.